### PR TITLE
Add logging options to virtual workspaces server

### PR DESCRIPTION
**Pull request by csams**
## Summary
Add logging options to the virtual workspaces server.

Signed-off-by: Christopher Sams <csams@redhat.com>

### CODE REVIEW

File: cmd/virtual-workspaces/command/cmd.go [hunk 1]
```diff
@@ -60,6 +60,10 @@ func NewCommand(out, errout io.Writer, stopCh <-chan struct{}) *cobra.Command {
 		Long:  "Start a virtual workspace apiserver to managing personal, shared or organization workspaces",
 
 		Run: func(c *cobra.Command, args []string) {
+			if err := opts.Logs.ValidateAndApply(); err != nil {
+				klog.Fatal(err)
```
Comment (sttts) [inline cmd/virtual-workspaces/command/cmd.go:64]: this print stack trace. Better print the error normally, and then os.Exit

Comment (sttts) [reply cmd/virtual-workspaces/command/cmd.go:64]: better: use the RunE variant for the command and just return the error, also below

Comment (csams) [reply cmd/virtual-workspaces/command/cmd.go]: good call. I converted to RunE but kept the block that prints details if the error was invalid. Is that still useful, or should I remove it, too?

Comment (sttts) [reply cmd/virtual-workspaces/command/cmd.go]: honestly I don't understand why that code is there. Would remove it.

Comment (csams) [reply cmd/virtual-workspaces/command/cmd.go]: Done

**Review by sttts — APPROVED** [review]

**Pull request merged**